src/NzbDrone.Core.Test/NotificationTests/Xbmc/Http/UpdateFixture.cs

```diff
@@ -28,47 +28,47 @@ public void Setup()
                 UpdateLibrary = true
             };
 
-            _fakeSeries = Builder<Artist>.CreateNew()
-                                         .With(s => s.ForeignArtistId = "79488")
+            _fakeArtist = Builder<Artist>.CreateNew()
+                                         .With(s => s.ForeignArtistId = "123d45d-d154f5d-1f5d1-5df18d5")
```

Comment:
I don't know that this will matter, but I don't think this is the format of UUID we have. For example, one musicbrainz artist is `9f4e41c3-2648-428e-b8c7-dc10465b49ac`.

Reply:
You are right, I copied from another test, will fix